packages/agoric-cli/src/commands/inter.js

```diff
@@ -64,34 +65,38 @@ const makeFormatters = assets => {
     const [dl, dm] = fmtAmtTuple(br(r.denominator));
     return `${Number(nm) / Number(dm)} ${nl}/${dl}`;
   };
+  /** @param {Ratio} r */
   const discount = r =>
     100 - (Number(r.numerator.value) / Number(r.denominator.value)) * 100;
-  return { amount, record, price, discount };
-};
 
-/**
- * Format amounts in vaultManager metrics for JSON output.
- *
- * @param {*} metrics manager0.metrics
- * @param {*} quote manager0.quote
- * @param {*} assets agoricNames.vbankAssets
- */
-const fmtMetrics = (metrics, quote, assets) => {
-  const fmt = makeFormatters(assets);
-  const { liquidatingCollateral, liquidatingDebt } = metrics;
+  /** @param {import('@agoric/time/src/types.js').TimestampRecord} tr */
```

Comment:
I think these types are exported at package level
```suggestion
  /** @param {import('@agoric/time').TimestampRecord} tr */
```

Reply:
Thanks... I just accepted what I got from ctrl-space

Reply:
darn... ci is not happy:

```
Error: src/commands/inter.js(65,38): error TS2694: Namespace '"/home/runner/work/agoric-sdk/agoric-sdk/packages/time/index"' has no exported member 'TimestampRecord'.
```